DIOP.Analysis.Draft/main.tex

```diff
@@ -0,0 +1,266 @@
+         Fair& 4,816\\
+         Poor& 965\\
+         Don't Know& 16\\
+         Refused& 2\\
+         Total& 19,970\\
+    \end{tabular}
+    \caption{Caption}
+    \label{tab:my_label}
+\end{table}
+
+
+-----------------------------------------
+                             |  Frequency
+-----------------------------+-----------
+compare mem to prev wave- pc |           
+  1                          |          6
+  2                          |        163
+  3                          |         48
+  9                          |          1
+  Total                      |        218
+-----------------------------------------
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{frequency_histogram_pz216.png}
+    \caption{Enter Caption}
+    \label{fig:enter-label}
+\end{figure}
+
+
+Describe your data. Where you got it from, how it was generated, what variables you'll use, what data cleaning steps you had to take, where your processed data, code and documentation is stored.
+
+In a published paper, a lot of this detail will be in a data appendix. For the purposes of this report, include it all here (this may be the longest section of your report).!
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{ever had dementia vs r college degree ctab.doc.pdf}
+    \caption{Ever had dementia (PC273) vs college degree (PB016)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{ever had dementia vs r high school diploma or ged ctab.doc.pdf}
+    \caption{Ever had dementia (PC273) vs college degree (PB015)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{frequency_histogram_pz216.png}
+    \caption{Frequency Histogram of PZ216 (r years of education)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{pd554_over_pc273.png}
+    \caption{PD554 (get lost in familiar places) over PC273 (ever had dementia)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{pd554_over_pz216.png}
+    \caption{PD554 (get lost in familiar places) over PZ216 (r years of education)
+}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{pv009_over_pc273.png}
+    \caption{PV009 (forgetful during daily activities) over PC273 (ever had dementia)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{pv009_over_pz216.png}
+    \caption{PV009 forgetful during daily activities) over PZ216 (r years of education)}
+    \label{fig:enter-label}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\linewidth]{pz261_over_pz216.png}
+    \caption{PZ261 (pw Alzheimer's) over PZ216 (r years of education)
```

Comment:
What do we learn from these graphs? I'm not sure what I'm looking at in this one or the following one.